### ISSUE TITLE
BNAndPadLayer may cause no padding problem in SyncBatchnorm mode

### ISSUE THREAD

**Issue by ramosmy** [DingXiaoH/DiverseBranchBlock#8]
Just turn all batchnorm layer in DBB to sync mode in pytorch  and BNAndPadLayer will cause no padding operations.
